Dokan/DokanTests/Scenes/ReviewProduct/ReviewProductViewControllerTests.swift

```diff
@@ -0,0 +1,113 @@
+//
+//  ReviewProductViewControllerTests.swift
+//  DokanTests
+//
+//  Created by marwa on 31/08/2022.
+//
+
+@testable import Dokan
+import XCTest
+
+class ReviewProductViewControllerTests: XCTestCase {
+    // MARK: - Properties
+
+    var sut: ReviewProductViewController!
+    var viewModel: ReviewProductViewModel!
+
+    // MARK: - life cycle
+
+    override func setUpWithError() throws {
+        viewModel = ReviewProductViewModel()
```

Comment:
You should make `MockReviewProductViewModel` so you don't make a real requests and return mock data.